src/languages/nim/index.ts

```diff
@@ -96,7 +96,7 @@ const nimLanguage: Language = {
       useMultireplace(),
       inlineVariables,
       forArgvToForEach,
-      forArgvToForRange(),
+      forArgvToForRange(true),
```

Comment:
`forArgvToForRange(true, true)` ?

Reply:
With the `forRangeToForRangeInclusive` plugin included, it should not matter.